Artifacts.toml

```diff
@@ -0,0 +1,6 @@
+[CUDA_lib]
+git-tree-sha1 = "2918fba865582556e219191a7f393c47c2e822e0"
+
+    [[CUDA_lib.download]]
+    sha256 = "751bf9d1f2921d4176ffb8ed1ddbd59bb60d6a517e6784bb71d61b62357c0007"
+    url = "https://gist.github.com/ArrogantGao/c38791f143d36d4b2481ac7e4aa4ecce/raw/2918fba865582556e219191a7f393c47c2e822e0.tar.gz"
```

Comment:
I think since we haven't decided yet whether to keep this version in the future, perhaps we can preserve using gist for now? If we decide to use it later, I can package it together using BinaryBuilder.

Reply:
Sounds good.